pex/variables.py

```diff
@@ -237,15 +262,17 @@ def PEX_MODULE(self):
 
     @property
     def PEX_PROFILE(self):
+        # type: () -> Optional[bool]
         """Boolean.
 
         Enable application profiling.  If specified and PEX_PROFILE_FILENAME is not specified, PEX
         will print profiling information to stdout.
         """
-        return self._get_path("PEX_PROFILE", default=None)
+        return self._get_bool("PEX_PROFILE", default=False)
```

Comment:
This was confusing to me. I think a bug. We document it as a `bool`, but treat it as a `path`. @jsirois lmk if you know what we intended.